texar/torch/modules/encoder_decoders/t5_encoder_decoder.py

```diff
@@ -0,0 +1,379 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
```

Comment:
Can we have a general encoder decoder class instead of this? The current implementation doesn't seem to use any encoder/decoder-specific features.

Reply:
Right because it is a raw T5 pretrained model as of now.